modules/api/src/main/java/net/countercraft/movecraft/craft/CraftType.java

```diff
@@ -316,6 +317,8 @@ public CraftType(File f) {
         int dropdist = integerFromObject(data.getOrDefault("gravityDropDistance", -8));
         gravityDropDistance = dropdist > 0 ? -dropdist : dropdist;
         collisionSound = Sound.valueOf((String) data.getOrDefault("collisionSound", "BLOCK_ANVIL_LAND"));
+        disableTeleportToWorlds = stringListFromObject(data.getOrDefault("disableTeleportToWorlds", new ArrayList<>()));

```

Comment:
I defer to CC's knowledge on the subject, but is an ArrayList the best data structure for this?

Reply:
stringListFromObject converts a List to a string array

Reply:
I would store it in a set after, as the main use of `disableTeleportToWorlds` is checking if an element is inside it